### ISSUE TITLE
DeprecationWarning: getargs: The 'u' format is deprecated. Use 'U' instead.

### ISSUE THREAD

**Issue by juliangilbey** [jamesturk/jellyfish#164]
The following minimal example gives this warning when run under Python 3.10.6:
```python
import jellyfish
print(jellyfish.damerau_levenshtein_distance("01", "10"))
```
It doesn't give the warning when `jaro_distance` is used instead.

**Comment by NickCrews** [comment]
I believe this was fixed by https://github.com/jamesturk/cjellyfish/pull/12, which was pulled into this repo at https://github.com/jamesturk/jellyfish/commit/0ba387a2b7037bcf4898af75fa3ce27515bd5fc7, and so should be included in releases 0.9.0 and greater? (I might not be interpreting this chain of events correctly)

@juliangilbey do you see this error when running on the latest version?

**Comment by juliangilbey** [comment]
Ah, oops, I hadn't noticed that the Debian version of jellyfish is behind the upstream version.  So I'll close this issue - sorry for the noise.
